### PR TITLE
AS4 Upgrade

### PR DESCRIPTION
**Pull request by mabuyo**
Upgrades to use Apollo Server 4.

DO NOT MERGE until course content is also updated.

### CODE REVIEW

File: final/subgraph-locations/index.js [hunk 1]
```diff
@@ -1,28 +1,38 @@
-const { ApolloServer, gql } = require('apollo-server');
-const { readFileSync } = require('fs');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
 
+const { readFileSync } = require('fs');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline final/subgraph-locations/index.js:6]:
```suggestion
const gql = require('graphql-tag');
```

File: final/subgraph-reviews/index.js [hunk 2]
```diff
@@ -1,28 +1,38 @@
-const { ApolloServer, gql } = require('apollo-server');
-const { readFileSync } = require('fs');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
 
+const { readFileSync } = require('fs');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline final/subgraph-reviews/index.js:6]:
```suggestion
const gql = require('graphql-tag');
```

File: subgraph-locations/index.js [hunk 3]
```diff
@@ -1,28 +1,35 @@
-const { ApolloServer, gql } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
+
 const { readFileSync } = require('fs');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline subgraph-locations/index.js:5]:
```suggestion
const gql = require('graphql-tag');
```

File: subgraph-reviews/index.js [hunk 4]
```diff
@@ -1,28 +1,35 @@
-const { ApolloServer, gql } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
+
 const { readFileSync } = require('fs');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline subgraph-reviews/index.js:5]:
```suggestion
const gql = require('graphql-tag');
```

**Review by lizhennessy — APPROVED** [review]
✅ starter code works great
✅ final folder code works great

**Review by lizhennessy — APPROVED** [review]

**Pull request merged**